import_video.py

```diff
@@ -66,8 +63,9 @@ def get_params(self, path):
                     lng = float(match.group('lng'))
                 except: pass
         if not camera:
-            logging.warn('did not detect camera name, assuming "unnamed"')
-            camera = DEFAULT_CAMERA_NAME
+            logging.error('did not detect camera name, assuming "%s"', self.DEFAULT_CAMERA_NAME)
+            sys.exit(1)
+            camera = self.DEFAULT_CAMERA_NAME
```

Comment:
It's confusing that the variable name is "camera" when it's assigned only a string rather than an object like {"name":"unnamed"}, so rename this variable to camera_name or make "camera" an object with the attribute name.